File: SZIntraV3_1_WebSite/QDT/app/model/ll/Lesson.js
/*global _comma_separated_list_of_variables_*/
Ext.define('QDT.model.ll.Lesson', {
    extend: 'Ext.data.Model',
    requires: [
        'Asz.util.UserProfile'
    ],
    fields: [
        'id',
        'category_id',
        'subject',
        'detail',
        'owner_id',
        'business',
        'restrict_all',
        'create_date',
        'create_by',
        'working_group',
        'effective_time',
        'status',
        'skill_code_binding_mode',
        'learning_cycle',
        'failure_mode',
        'attachment_quantity',
        {
            name: 'detail_title', convert: function (v, record) {
                return Profile.getText('Lessons') + ' - ' + record.data.id;
            }
        }
    ]
});

File: SZIntraV3_1_WebSite/Models/Ll/Lesson.cs
using System;
using System.Collections.Generic;
using System.Dynamic;
using System.Linq;
using System.Web;
using Org.BouncyCastle.Utilities;

namespace SZIntraV3_1_WebSite.Models.Ll
{
    public class Lesson
    {
        public int id { get; set; }
        public int category_id { get; set; }
        public string subject { get; set; }
        public string detail { get; set; }
        public int owner_id { get; set; }
        public string business { get; set; }
        public DateTime create_date { get; set; }
        public int create_by { get; set; }
        public int attachment_quantity { get; set; }
        public string working_group { get; set; }
        public bool? restrict_all{get; set;}
        public int? effective_time { get; set; }
        public string status { get; set; }
        public int? failure_mode { get; set; }
        public int? learning_cycle { get; set; }

        public string skill_code_binding_mode { get; set; }

        public string part_num { get; set; }
        public int? oper_num { get; set; }
    }
}